test/integration/network-e2e/sample-transaction-event-handler.js

```diff
@@ -144,11 +144,12 @@ class SampleTransactionEventHandler {
  * @param {Transaction} transaction The transaction being submitted.
  * @param {Network} network The network where the transaction is being submitted.
  */
-function createTransactionEventHandler(transaction, network) {
+function createTransactionEventHandler(transaction, options) {
```

Comment:
The JSDoc for this function (the lines immediately above) need to be updated to match the parameters